Robus/inc/reception.h

```diff
@@ -19,9 +19,10 @@ typedef struct __attribute__((__packed__))
     {
         struct __attribute__((__packed__))
         {
-            unsigned char identifier : 4;
-            unsigned char rx_error : 1;
-            unsigned char rx_timeout : 1;
+            uint8_t identifier : 4;
+            uint8_t rx_error : 1;
+            uint8_t rx_timeout : 1;
+            uint8_t rx_framing_error : 1;
```

Comment:
Watch out this is not the bug but an RX_framing_error allowing to prevent collision errors.

Reply:
And by the way I think there is multiple bugs on this PR you should perhaps put `bug` into `bugs` on the commit msg.